entries/103_show_notes.bacf

```diff
@@ -0,0 +1,41 @@
+- nr: 103
+- title: A Philosophical Treatise on the Definition of Pro
+- date: 06.09.18, 20:00
+- file: 103_a_philosophical_treatise_on_the_definition_of_pro.mp3
+- duration: 00:32:26
+- length: TO BE ADDED
```

Comment:
We should not merge before adding this; the direct download I used was about 20mb and I think that was not going to be the final size. @terhechte